m/mink-selenium2-driver/minkseleniumdriver_ubi8.5.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -e
+
+# ----------------------------------------------------------------------------
+# Package          : minkseleniumdriver
+# Version          : latest
```

Comment:
this should be the same as the version we are actually building in the script, v6.2.0 in this case?